docs/tutorials/rerank/rerank_pipeline_with_Cohere_Rerank_model.md

```diff
@@ -0,0 +1,306 @@
+# Topic
+
+Rerank Pipeline is a feature released in OpenSearch 2.12. It can rerank the search result with the relevance score between search query and each document in search result.
```

Comment:
```suggestion
Reranking pipeline is a feature released in OpenSearch 2.12. It can rerank search results, providing a relevance score for each document in the search results with respect to the search query.
```